File: PalindromeNumber/palindromeNumber.js
// https://leetcode.com/problems/palindrome-number/

function palindromeNumber(number) {
    let reverseNumber = '';
    number = number.toString();
    for (let i = number.length - 1; i > -1; i--) {
        reverseNumber += number[i];
    }
    console.log(reverseNumber);
    return number === reverseNumber;
}

console.log(palindromeNumber(1234321));
console.log(palindromeNumber(4569874));

File: PalindromeNumber/palindrome_number.py
# https://leetcode.com/problems/palindrome-number/

class Solution:
    def palindrome_number(self, number):
        number = str(number)
        reverse_number = ''
        for i in range(len(number) - 1, -1, -1):
            reverse_number += number[i]
        return number == reverse_number

if __name__ == '__main__':
    solution = Solution()
    print(solution.palindrome_number(int(input('Enter a number : '))))
